operator/main.go

```diff
@@ -41,7 +43,6 @@ import (
 	resourcesv5 "github.com/gravitational/teleport/operator/apis/resources/v5"
 	resourcescontrollers "github.com/gravitational/teleport/operator/controllers/resources"
 	"github.com/gravitational/teleport/operator/sidecar"
-	//+kubebuilder:scaffold:imports
```

Comment:
Was this intentional?

Reply:
Not really, this was done by something I ran to get the linter to pass (either gci or goimports). Just reverting to master all of the files I touched incidentally from those runs: 5834a1f41892f74faeba8f7b58ad4505b24c8983